app/helpers/pg_hero/home_helper.rb

```diff
@@ -26,5 +26,15 @@ def pghero_remove_index(query)
       ret << ", column: #{columns.inspect}" if columns
       ret
     end
+
+    def formatted_vacuum_times(time)
+      content_tag(:span, title: formatted_date_time(time)) do
+        "#{time_ago_in_words(time.in_time_zone(@time_zone))} ago"
+      end
+    end
+
+    def formatted_date_time(time)
+      l time.in_time_zone(@time_zone), format: "%d %b %Y %H:%M"
```

Comment:
Think we can probably use `:long` since it no longer needs to fit into the table cell.